Runtime/McSubscriber.cs

```diff
@@ -508,6 +553,35 @@ public void RemoveRenderAudioSource(AudioSource source)
         {
             _renderer.RemoveAudioTarget(source);
         }
+        private Layer GetSimulcastlayer(string rid)
+        {
+            foreach (var item in simulCastInfo.Layers)
+            {
+                if(rid.ToLower().Equals(item.EncodingId.ToLower()))
+                    return item;
+            }
+            return null;
+        }
+        /// <summary>
+        /// Set a simulcast layer
+        /// </summary>
+        /// <param name="layer">
+        public void SetSimulcastLayer(string simulcastId)
```

Comment:
This needs to take a whole Layer instead, as encoding Id is not a unique identifier